### PR TITLE
Creates the install guide under filename CONTRIBUTING.md

### LINKED ISSUE #29: Install guide
We should give steps to clone the polyfill, and to build and run the tests.

Perhaps assume the reader starts with only a new Goobuntu install.

### PR DESCRIPTION
**Pull request by courtneycb**
Resolves #29

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+$ npm install  
+$ npm install -g grunt grunt-cli  
+
+**Build and run tests**  
+$ npm test  
+To watch for file changes and automatically retest, use:  
+$ npm run watchTests  
+
+**Use the polyfill**  
+Add third_party/web-animations-js/web-animations.dev.js and motion-path-polyfill.min.js to your web page.
```
Comment (alancutter) [inline CONTRIBUTING.md]: Point to a demo page as an example.

Comment (courtneycb) [inline CONTRIBUTING.md]: Is the way I added it okay?

Comment (alancutter) [inline CONTRIBUTING.md]: I'd make it a relative link instead.

Comment (courtneycb) [inline CONTRIBUTING.md]: like this motion-path-js/examples/offsetAnchorPosition.html?

Comment (alancutter) [inline CONTRIBUTING.md]: Not sure, you can test if it works using GitHub's file view.

**Pull request merged**